lib/danger/request_source/github.rb

```diff
@@ -231,6 +231,10 @@ def process_markdown(violation)
         message = match.nil? ? html : match.captures.first
         Violation.new(message, violation.sticky)
       end
+
+      def organisation
```

Comment:
Agreed - it's hard coded and not documented yet, since I wasn't sure if that's the right approac